admin/model_region_config.go

```diff
@@ -4,198 +4,370 @@ package admin
 
 import (
 	"encoding/json"
-	"fmt"
 )
 
-// RegionConfig - Cloud service provider on which MongoDB Cloud provisions the hosts.
+// checks if the RegionConfig type satisfies the MappedNullable interface at compile time
+var _ MappedNullable = &RegionConfig{}
+
+// RegionConfig Cloud service provider on which MongoDB Cloud provisions the hosts.
 type RegionConfig struct {
-	AWSRegionConfig    *AWSRegionConfig
-	AzureRegionConfig  *AzureRegionConfig
-	GCPRegionConfig    *GCPRegionConfig
-	TenantRegionConfig *TenantRegionConfig
+	AnalyticsAutoScaling *AutoScalingV15        `json:"analyticsAutoScaling,omitempty"`
```

Comment:
Documentation for the merged object looks very good. It doesn't have any AWS, GCP specific comments as descriptions are missing from the fields that are merged.